src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -21,7 +22,11 @@
  * The main LogicManager of the app.
  */
 public class LogicManager implements Logic {
-    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
+    public static final String FILE_OPS_ERROR_FORMAT = "Could not save data due to the following error: %s";
+
+    public static final String FILE_OPS_PERMISSION_ERROR_FORMAT =
+            "Could not save data to file %s due to insufficient permissions to write to the file or the folder.";
```

Comment:
```suggestion
            "Could not perform write operations to file %s due to insufficient permissions";
```

Reply:
I wouldn't mind keeping the original, as it is more informative and context-specific.

Reply:
I think the original feels more specific to the actual warning as well, `perform write operations` might be a bit too generic.